src/core_codemods/sonar/sonar_jwt_decode_verify.py

```diff
@@ -0,0 +1,43 @@
+import libcst as cst
+from codemodder.codemods.base_codemod import Reference
+from codemodder.result import same_line, fuzzy_column_match
+from codemodder.codemods.sonar import SonarCodemod
+from codemodder.codemods.libcst_transformer import (
+    LibcstTransformerPipeline,
+)
+from core_codemods.jwt_decode_verify import JwtDecodeVerify, JwtDecodeVerifyTransformer
+
+
+class JwtDecodeVerifySonarTransformer(JwtDecodeVerifyTransformer):
+    def filter_by_result(self, node) -> bool:
+        """
+        Special case result-matching for this rule because the sonar
+        results returned have a start/end column for the verify keyword
+        within the `decode` call, not for the entire call like semgrep returns.
+        """
+        if self.results is None:
```

Comment:
I believe this will actually never occur in the Sonar case and you can remove this.

Reply:
yep very good point. I was trying to keep this method as similar to its base one, but you're right it's nonsensical in this case.